nl_server/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
Can you do a search for embeddings.yaml, and ensure references are fixed?

Reply:
all in the build custom dc tool comments. I will update them when I get to change that tool.